src/main/java/io/anserini/collection/NewYorkTimesCollection.java

```diff
@@ -47,25 +50,22 @@
 import java.util.Date;
 import java.util.HashSet;
 import java.util.List;
-import java.util.NoSuchElementException;
 import java.util.Set;
 
 /**
  * Class representing an instance of the New York Times Annotated Corpus,
  * <a href="https://catalog.ldc.upenn.edu/products/LDC2008T19">LDC2008T19</a>.
  * Note that the collection is distributed as a number of {@code tgz} files, which
- * uncompresses to individual XML documents in a directory structure. Since the
- * current design of {@link io.anserini.collection.Collection.FileSegment} cannot
- * handle {@code tgz} files (only {@code gz} files), the collection must first be
- * uncompressed prior to indexing. In this case, each {@code FileSegment} is an
- * XML file containing only a single document.
+ * uncompresses to individual XML documents in a directory structure. Currently
+ * {@code FileSegment} can take either a {@code tgz} file or a {@code xml} file.
+ * If the {@code FileSegment} contains is an XML file, it contains only a single document.
  */
 public class NewYorkTimesCollection extends Collection<NewYorkTimesCollection.Document> {
   private static final Logger LOG = LogManager.getLogger(NewYorkTimesCollection.class);
 
   @Override
   public List<Path> getFileSegmentPaths() {
-    Set<String> allowedFileSuffix = new HashSet<>(Arrays.asList(".xml"));
+    Set<String> allowedFileSuffix = new HashSet<>(Arrays.asList(".xml", ".tgz"));
```

Comment:
I think we want to remove `".xml"` here, right?

Reply:
My current design actually supports both `tgz` and `xml` files to be indexed, just in case.